src/api/routers/participants/participantsRouter.ts

```diff
@@ -217,6 +220,71 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.put('/', isApproverCheck, async (req, res) => {
+    const participantRequest = AdminParticipantCreationPartial.parse(req.body);
```

Comment:
I wonder if the logic of this function could be put into a new file `participantsApproval.ts` or similar - or at least `participantsService`. I'd be nice to make the router as thin as possible, and it also makes it easier to test using the pattern established in https://github.com/IABTechLab/uid2-self-serve-portal/pull/275

Reply:
I moved it to participantCreation.ts